install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -90,9 +90,10 @@ instructions.
 . Provision as many new machines as there are masters to replace.
 
 . Add or expand the cluster. for example, if you want to add 3 masters with etcd
-colocated, scale up 3 master nodes or 3 etcd nodes.
+colocated, scale up 3 master nodes.
```

Comment:
Same here - if etcd is added automatically, this process needs to change.

Reply:
@gpei, is there a separate process that we should consider for the initial release?

Reply:
I added a note at line 95.

Reply:
Etcd scale-up using playbook is the same with previous release, no extra separate process needed.

Reply:
Great. Thank you!